pkg/collector/corechecks/gpu/nvidia/helpers.go

```diff
@@ -79,14 +80,23 @@ func filterSupportedAPIs(device ddnvml.Device, apiCalls []apiCallInfo) []apiCall
 
 // GetDeviceTagsMapping returns the mapping of tags per GPU device.
 func GetDeviceTagsMapping(deviceCache ddnvml.DeviceCache, tagger tagger.Component) map[string][]string {
-	devCount := deviceCache.Count()
+	devCount, err := deviceCache.Count()
+	if err != nil {
+		log.Warnf("Error getting device count: %s", err)
```

Comment:
i think we have a global log limiter for the collectors, so we can use to avoid spamming